packages/confirmation-modal/src/ConfirmationModal.spec.tsx

```diff
@@ -119,10 +119,10 @@ describe('packages/confirmation-modal', () => {
         requiredInputText: 'Confirm',
       });
 
-      const confirmationButton = getByText('Confirm');
+      const confirmationButton = getByText('Confirm').parentNode;
```

Comment:
I propose doing `getByText('Confirm').closest('button')` to make this more robust

Reply:
👍